dockers/docker-dhcp-relay/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM docker-base
+
+## Make apt-get non-interactive
+ENV DEBIAN_FRONTEND=noninteractive
+
+## Install isc-dhcp-relay
+## Clean up
+RUN apt-get update && apt-get -y install \
+        isc-dhcp-relay &&		 \
+	apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
+
+COPY isc-dhcp-relay /etc/default/isc-dhcp-relay
+
+ENTRYPOINT service isc-dhcp-relay start \
+	&& /bin/bash
```

Comment:
do we need to start rsyslog in the docker? how do we provide logging functionaility?